src/lib/server/editors/NoNegationEditor.ts

```diff
@@ -13,55 +47,111 @@ export class NoNegationEditor extends PromptEditor {
 		);
 	}
 
-	async filter(prompt: Tables<'prompts'>, llm: LLM): Promise<boolean> {
-		const systemPrompt = `### Task
-You are an AI prompt writing critiquer. Your task is to determine if a prompt tells a model what **not** to instead of what it should do. 
-	
-### Instructions
-* If the prompt only tells the model what it should do, return true.
-* If the prompt instead tells the model what it should **not** do, return false.
-	 
-### Output
-Return the output in JSON with the key "output" that is either true or false.`;
-		const res = await llm.generate(
-			[
-				{
-					role: 'system',
-					content: systemPrompt
-				},
-				{
-					role: 'user',
-					content: prompt.prompt
-				}
-			],
-			{ json: true, temperature: 0 }
+	async canBeApplied(prompt: Tables<'prompts'>, llm: LLM) {
+		const regex = /([^.?!]+[.?!])/g; // Regex to match sentences including the punctuation
```

Comment:
Do we run into issues with floating point numbers like DROP?

Reply:
Hmm probably. Thoughts on a better solution? Best I can think of for now... We could ask the LLM to do the sentence splitting but seems more risky aha.